docs/version0.36.md

```diff
@@ -28,6 +28,7 @@ The following new features and notable changes since version 0.35.0 are included
 
 - [New binaries and changes to supported environments](#binaries-and-supported-environments)
 - [New `-XX:JITServerAOTmx` option added](#new-xxjitserveraotmx-option-added)
+- [Changes to the location of the default shared cache and cache snapshot directory](#changes-to-the-location-of-the-default-shared-cache-and-cache-snapshot-directory)
```

Comment:
I suggest this title should be reworded from "default shared cache". The default shared cache directory is changed, which shouldn't be confused with the shared cache created by default. Maybe "Changes to the default directory for the shared cache and snapshot".